RUFAS/routines/animal/life_cycle/cow.py

```diff
@@ -141,6 +143,73 @@ def __init__(self, args):
             self.set_parity_index()
             self.set_lactation_curve_params()
 
+    def get_cow_values(self) -> Dict[str, Any]:
+        return {
+            "id": self.id,
+            "breed": self.breed,
+            "birth_date": self.birth_date,
+            "days_born": self.days_born,
+            "birth_weight": self.birth_weight,
+            "body_weight": self.body_weight,
+            "wean_weight": self.wean_weight,
+            "events": str(self.events),
+            "repro_program": self.repro_program,
+            "tai_method_h": self.tai_method_h,
+            "synch_ed_method_h": self.synch_ed_method_h,
+            "mature_body_weight": self.mature_body_weight,
+            "estrus_count": self.estrus_count,
+            "estrus_day": self.estrus_day,
+            "tai_program_start_day_h": self.tai_program_start_day_h,
+            "synch_ed_program_start_day_h": self.synch_ed_program_start_day_h,
+            "synch_ed_estrus_day": self.synch_ed_estrus_day,
+            "synch_ed_stop_day": self.synch_ed_stop_day,
+            "conception_rate": self.conception_rate,
+            "ai_day": self.ai_day,
+            "abortion_day": self.abortion_day,
+            "days_in_preg": self.days_in_preg,
+            "gestation_length": self.gestation_length,
+            "p_gest_for_calf": self.p_gest_for_calf,
+            "calf_birth_weight": self.calf_birth_weight,
+            "presynch_method": self.presynch_method,
+            "tai_method_c": self.tai_method_c,
+            "resynch_method": self.resynch_method,
+            "days_in_milk": self.days_in_milk,
+            "parity": self.calves,
+            "calving_interval": self.CI
+        }
```

Comment:
I don't know how the encapsulation needs to work for `Cow`, but all tests pass when this suggestion is applied. I think it is also a little more future-proof, in case the attributes in `Cow` change.
```suggestion
        return self.__dict__
```